macro/src/codegen.rs

```diff
@@ -744,6 +857,13 @@ impl ClassInfo {
             .collect()
     }
 
+    fn class_generic_names_with_defaults(&self) -> Vec<TokenStream> {
+        self.class_generic_names()
+            .into_iter()
+            .map(|g| quote_spanned!(self.span => #g = java::lang::Object))
```

Comment:
Interesting. The downside of this is that it gives us inaccurate return types, right?

Previously I was considering adding a special struct like `UnknownClass` or `Wildcard` and then translating `?` in return position to that struct. It's not *100%* correct -- but it's probably close enough for "government work". It would allow you to call Java methods that perhaps you ought not to be able to call (e.g., a method like `<T> (Class<T>, Class<T>)`), but is that a realistic danger?

Anyway, that seems preferable to using `java::lang::Object` here...? Maybe not always.

Reply:
Filed #57